### DISCUSSION TITLE
Retiring variations of using brackets as a denominator for footnotes

### DISCUSSION THREAD

**Discussion by markcheret** [markcheret/footnotes#136] [Ideas]
There have been a load of support tickets opened because of how footnotes are being denominated, especially those using ((brackets)).
Is there a safe way to parse those?
Or should we think about
- phasing them out over the next few releases (with an option to auto-convert to another format)
- move the default setting for new installations to something like [fn]footnote[/fn]
- both?

**Comment by Rumperuu** [comment]
Currently, I think changing the default to something else would be a good short-term fix. `[fn]`/`[/fn]` is unlikely to clash with anything, but I do feel like part of the appeal of the Plugin is the simplicity of the `((`/`))`, which is lost somewhat with that. That said, Easy Footnotes uses the relatively cumbersome `[efn_note]` and that doesn't seem to have done it any harm.

As we currently don't do any static parsing of posts, auto-conversion would be quite a bit of work to implement (but it may be able to piggyback off of a feature like #107 if/when we get around to implementing it). However, #121 should mitigate a lot of the `((`/`))` conflict issues because DOM parsing will solve things like the Plugin detecting footnotes inside `<script>` tags, etc.
